olap/src/aggregator/mod.rs

```diff
@@ -173,36 +175,12 @@ async fn process_feed_batch(ctx: SessionContext, conclusion_feed: DataFrame) ->
         )?
         // Project joined columns to just the ones we need
         .select(vec![
-            col(Column {
-                relation: Some(TableReference::from("?table?")),
-                name: "index".to_string(),
-            })
-            .alias("index"),
-            col(Column {
-                relation: Some(TableReference::from("?table?")),
-                name: "event_type".to_string(),
-            })
-            .alias("event_type"),
-            col(Column {
-                relation: Some(TableReference::from("?table?")),
-                name: "stream_cid".to_string(),
-            })
-            .alias("stream_cid"),
-            col(Column {
-                relation: Some(TableReference::from("?table?")),
-                name: "controller".to_string(),
-            })
-            .alias("controller"),
-            col(Column {
-                relation: Some(TableReference::from("?table?")),
-                name: "dimensions".to_string(),
-            })
-            .alias("dimensions"),
-            col(Column {
-                relation: Some(TableReference::from("?table?")),
-                name: "event_cid".to_string(),
-            })
-            .alias("event_cid"),
+            col("conclusion_feed.index").alias("index"),
```

Comment:
Thanks for this. Did the exact same thing in my test haha. Was getting an ambiguous reference error!